netci.groovy

```diff
@@ -2871,53 +2942,130 @@ Constants.allScenarios.each { scenario ->
                                     shell("${dockerCmd}./tests/scripts/run-gc-reliability-framework.sh ${architecture} ${configuration}")
                                 }
                             } 
-
                             else { // windowsArmJob == true
-                                // Unzip tests.
-                                batchFile("powershell -NoProfile -Command \"Add-Type -Assembly 'System.IO.Compression.FileSystem'; [System.IO.Compression.ZipFile]::ExtractToDirectory('bin\\tests\\tests.zip', 'bin\\tests\\${osGroup}.${architecture}.${configuration}')")
                                 
-                                // Build the build commands
-                                def buildCommands = ""
-                                
-                                def coreRootLocation = "%WORKSPACE%\\bin\\tests\\Windows_NT.${architecture}.${configuration}\\Tests\\Core_Root"
-                                def addEnvVariable =  { variable, value -> buildCommands += "set ${variable}=${value}\r\n"}
-                                def addCommand = { cmd -> buildCommands += "${cmd}\r\n"}
+                                if (isCoreFxScenario(scenario)) {
+
+                                    // Only arm supported for corefx testing now.
+                                    assert architecture == 'arm'
+
+                                    // Unzip CoreFx runtime
+                                    batchFile("powershell -NoProfile -Command \"Add-Type -Assembly 'System.IO.Compression.FileSystem'; [System.IO.Compression.ZipFile]::ExtractToDirectory('_\\fx\\fxruntime.zip', '_\\fx\\bin\\testhost\\netcoreapp-Windows_NT-Release-arm')")
+
+                                    // Unzip CoreFx tests.
+                                    batchFile("powershell -NoProfile -Command \"Add-Type -Assembly 'System.IO.Compression.FileSystem'; [System.IO.Compression.ZipFile]::ExtractToDirectory('_\\fx\\fxtests.zip', '_\\fx\\bin\\tests')")
+
+                                    // Add the script to run the corefx tests
+                                    def corefx_runtime_path   = "_\\fx\\bin\\testhost\\netcoreapp-Windows_NT-Release-arm"
+                                    def corefx_tests_dir      = "_\\fx\\bin\\tests"
+                                    def corefx_exclusion_file = "%WORKSPACE%\\tests\\arm\\corefx_test_exclusions.txt"
+                                    buildCommands = """\
+@echo off
```

Comment:
This needs to be in its own file that can be run and tested independently.

Reply:
Like a `tests\arm\run-corefx-tests.bat`?

Reply:
Generally these types of scripts go under tests/scripts